contracts/script/deploy/7683/DeployT1ERC7683Pull.s.sol

```diff
@@ -54,8 +55,8 @@ contract DeployT1ERC7683Pull is DeploymentUtils {
         vm.stopBroadcast();
     }
 
-    function t1_deploy() external {
-        logStart("DeployRouterPullBasedERC7683 to t1");
+    function t1_deploy(string calldata bridgePrefix) external {
```

Comment:
it would make more sense if we were passing the network name or chain ID instead. that provides more information than "L1" or "L2"